db.json

```diff
@@ -24,7 +24,7 @@
   "plannedTasks": [
```

Comment:
This file could be removed in this pull request if it is not used.
In addition, remove `json-server` from your project.